pkg/virt-controller/network/multus_annotations.go

```diff
@@ -56,18 +56,18 @@ func (mnap multusNetworkAnnotationPool) toString() (string, error) {
 	return string(multusNetworksAnnotation), nil
 }
 
-func GenerateMultusCNIAnnotation(namespace string, interfaces []v1.Interface, networks []v1.Network, config *virtconfig.ClusterConfig) (string, error) {
-	return GenerateMultusCNIAnnotationFromNameScheme(namespace, interfaces, networks, namescheme.CreateHashedNetworkNameScheme(networks), config)
+func GenerateMultusCNIAnnotation(namespace string, interfaces []v1.Interface, networks []v1.Network, networkToIPAMClaimName map[string]string, config *virtconfig.ClusterConfig) (string, error) {
```

Comment:
we cant because it means to read the NAD, which is read once at renderLaunchManifest
(fwiw it also has the vmi.Name in it)
anyhow lets please return to it once the other changes are done first